src/main/java/com/acrontum/spring/lib/AcrontumComponent.java

```diff
@@ -0,0 +1,40 @@
+package com.acrontum.spring.lib;
+
+import com.acrontum.spring.lib.security.JwtPrincipal;
+import java.util.Locale;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+/**
+ * The type Acrontum component.
+ */
+public class AcrontumComponent {
+
+    /**
+     * The Logger.
+     */
+    protected Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
It's not possible to use `@slf4j` because:
1. This class will be extended by other classes to provide general functionalities. 
2. If we use the annotation the generated code will be private and exclusive to the current like this:
`private static final org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(AcrontumComponent.class);`